CHANGELOG

```diff
@@ -4,8 +4,20 @@ qtile X.X.X, released XXXX-XX-XX:
 	- property "masterWindows" of Tile layout renamed to master_length
 	- properties wname, wmclass and role of Slice-layout replaced by Match-
 	  type property "match"
+        - rules specified in `layout.Floating`'s `float_rules` are now evaluated with
```

Comment:
Some whitespace issues here.

Reply:
The entire CHANGELOG seems to use a mix of tabs and spaces. I switched to using tabs now.

Reply:
So it is, we should probably fix that :)